### PR TITLE
Implement relaxed simd operations on ARM-64

### PR DESCRIPTION
**Pull request by zherczeg**
Added a commit which enables test to see they pass on CI. Will be removed since ARM-32 support is not present.

### CODE REVIEW

**Comment by zherczeg** [comment]
It looks like the owner of the docker files started many changes and it cause fails now:
https://github.com/uraimo/run-on-arch-action/commits/master/

I will try to figure out what to do.

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -158,7 +158,7 @@ jobs:
       with:
         submodules: true
     - name: Build in arm32 container
-      uses: uraimo/run-on-arch-action@v2.7.2
+      uses: uraimo/run-on-arch-action@master
```
Comment (clover2123) [inline .github/workflows/actions.yml:161]: New version **2.8.1** has been released just few hours ago (https://github.com/marketplace/actions/run-on-architecture).
What about using the latest version instead?

Comment (zherczeg) [reply .github/workflows/actions.yml]: Thank you for the idea. It is working.

**Review by clover2123 — APPROVED** [review]
LGTM

**Pull request merged**